### ISSUE TITLE
Author vs Contributor

### ISSUE THREAD

**Issue by Tardo** [OCA/odoo-community.org#38]
Hi, another day... another question xD

When you are an author and when you are an contributor?

-> Add new feature = Author
-> Migrate a module = Contributor
---> Migrate a module (rewriting: JS massive changes or something like that) = Author
-> Fix issues = Contributor
-> Add new module = Author

I'm in the correct way?

**Comment by elicoidal** [comment]
I do not think there is a formal definition anywhere and I think your approach makes sense.
In general my vision on that is that the author "brings new ideas" while the contributor "fosters them"

**Comment by pedrobaeza** [comment]
When implementing the "idea" requires a total rebuilding of the module in a new version while migrating it, the developer deserves the authorship in my opinion, so I agree on all the points brought by @Tardo.

**Comment by elicoidal** [comment]
Agree: this is all very subjective on the level of work/originality of the contributor.

Question is today there is no formal definition of the concept (AFAIK). Is it necessary or letting the users decide case by case (my preferred way) is good enough? 

Do we have often discussions on the topic?

**Comment by acysos** [comment]
Hello,

This is perfect, but like say @elicoidal, I don't see nothing formal. This is a great solution against people that they put as author for simple changes.

Greetings
